### PR TITLE
Update groups-teams-compliance-governance.md

### PR DESCRIPTION
**Pull request by msbemba**
A single retention policy can be set to apply to Teams chat, and Teams channel messages. When you add M365 Group, Teams chat and/or channel message cannot be selected.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4721

File: microsoft-365/solutions/groups-teams-compliance-governance.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ User chats are retained indefinitely even if a user account is deleted. If you d
 
 - [Retention policies in Microsoft Teams](/microsoftteams/retention-policies)
 
-A single retention policy can be set to apply to Microsoft 365 Groups, Teams chat, and Teams channel messages. 
+A single retention policy can be set to apply to Teams chat, and Teams channel messages.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/groups-teams-compliance-governance.md]:
```suggestion
A single retention policy can be set to apply to Teams chat and Teams channel messages. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**